floodviz/templates/index.html

```diff
@@ -3,7 +3,8 @@
 
 <head>
 	<meta charset="UTF-8">
-	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='bower_components/bootstrap/dist/css/bootstrap.css') }}"/>
+	<link rel="stylesheet" type="text/css"
```

Comment:
Stop changing this back and forth. This particular line has been change with almost every pull request.